aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -257,17 +263,35 @@ When using the out-of-process hosting model, the app might not shut down immedia
 ::: moniker-end
 
 ## Start-up error page
-
 ::: moniker range=">= aspnetcore-2.2"
 
-*Only applies to out-of-process hosting.*
+Both inprocess and outofprocess have custom error pages when they fail to start the application.
+
+If the ASP.NET Core module fails to find either the inprocess or out of process request handler, a *500.0 In-Process/Out-Of-Process Handler Load Failure* status code page appears.
+
+![502.5 Process Failure Status Code Page](aspnet-core-module/_static/ANCMV2-500-inproc.png)
+![502.5 Process Failure Status Code Page](aspnet-core-module/_static/ANCMV2-500-outofproc.png)
+
+For inprocess, if the ASP.NET Core module fails to start the application, a *500.30 Start Failure* status code page appears.
+
+![502.5 Process Failure Status Code Page](aspnet-core-module/_static/ANCMV2-500_30-inproc.png)
+
+For out of process, if the ASP.NET Core Module fails to launch the backend process or the backend process starts but fails to listen on the configured port, a *502.5 Process Failure* status code page appears.
+
+![502.5 Process Failure Status Code Page](aspnet-core-module/_static/ANCMV2-502_5-outofproc.png)
+
+To suppress this page and revert to the default IIS 5xx status code page, use the `disableStartUpErrorPage` attribute. For more information on configuring custom error messages,  [HTTP Errors `<httpErrors>`](/iis/configuration/system.webServer/httpErrors/).
```

Comment:
> For more information on configuring custom error messages,  [HTTP Errors `<httpErrors>`](/iis/configuration/system.webServer/httpErrors/).

* Add the word "see" right after the comma.
* There's an extra space here that should be 🔪 out
* Remove the code fencing within the link text and use gt and lt ...

Here's a cut='n-paste ...

> To suppress this page and revert to the default IIS 5xx status code page, use the \`disableStartUpErrorPage\` attribute. For more information on configuring custom error messages, see \[HTTP Errors \&lt;httpErrors\&gt;](/iis/configuration/system.webServer/httpErrors/).